net/respondd/src/respondd.c

```diff
@@ -649,10 +645,7 @@ int main(int argc, char **argv) {
 			break;
 
 		case 'd':
-			if (chdir(optarg)) {
```

Comment:
Without the `chdir`, we will to define where to store cache files in a different way.